css-text-4/Overview.bs

```diff
@@ -8157,10 +8157,17 @@ Inter-script Spacing</h4>
 	when they are directly adjoining on a line,
 	i.e. without any intervening non-zero [=margin=], [=border=], or [=padding=]
 	or intervening characters (such as a quotation mark or a space).
+	The amount of space introduced by these keywords is 1/8 of the CJK advance measure,
+	i.e ''0.125ic''.
 
 	Note: Spacing conventions vary, but values typically range from 1/4ic to as low as 1/8ic,
 	with 1/4ic being particularly common in monospace contexts
 	and 1/6ic being more common in proportional typesetting.
+	However because these spaces are inserted by default
```

Comment:
If you adopt the suggestion made in https://github.com/w3c/csswg-drafts/pull/8262/commits/aa252b9100d76945ae040e6c970780cc80d1f251#r1059249648, you could/should drop the "However" from this sentence.